puppet-theatre.gemspec

```diff
@@ -19,9 +19,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'sshkit', '1.9.0'
-  spec.add_dependency 'parallel', '1.10.0'
```

Comment:
These runtime dependencies should not be removed from gemspec.
Instead we should consider solving the problem by relaxing these version constraints. (I don't remember why I pinned the versions 😅 )